### DISCUSSION TITLE
How to fix typo's?

### DISCUSSION THREAD

**Discussion by mrtnbr** [bpatrik/pigallery2#740] [General]
During translation I have found some typo's and some text that would be more understandable is rephrased. I would not mind fixing this myself, but I don't know how. Do I just change the target text in the English translation file?  That's probably not the best way.

One typo that would need fixing before the 2.0.0 release would be the Option to fix the navbar on the screen is "Fix nabvar" (https://github.com/bpatrik/pigallery2/blob/2d6140e24a4159a144d5d6d8fa855f288f492f2e/src/frontend/app/ui/frame/frame.component.html#L169).

**Comment by bpatrik** [comment]
You will need to do the fix in the code. You can just search the code for
the English text.

Once text is updated in the code, someone should run the merge new
translation npm script (see package.json). I can do that if you do the typo
fixes.

-- Sorry for being brief, sent from my phone.
